python/ecs/fargate-service-with-efs/app.py

```diff
@@ -0,0 +1,127 @@
+from aws_cdk import (
+    aws_ec2 as ec2,
+    aws_ecs as ecs,
+    aws_efs as efs,
+    aws_iam as iam,
+    aws_ecs_patterns as ecs_patterns,
+    App, CfnOutput, Duration, Stack
+)
+from constructs import Construct
+class FargateEfs(Stack):
+
+    def __init__(self, scope: Construct, id: str, **kwargs) -> None:
+        super().__init__(scope, id, **kwargs)
+
+        #VPC
+        # Create VPC with 2 AZ and a Fargate Cluster
+        my_fargate_vpc = ec2.Vpc(
+            self, "my_fargate_vpc",
+            max_azs=2
+        )
+
+        # EFS
+        # File System Policy
+        my_file_system_policy = iam.PolicyDocument(
+            statements=[iam.PolicyStatement(
+                actions=["elasticfilesystem:ClientMount"],
+                principals=[iam.AnyPrincipal()],
```

Comment:
Instead of using iam.AnyPrincipal(), which is permissive, It might be better to restrict this to specific roles or users.